feature-11/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+    "key-1": {
+        "a": 1,
+        "b": 2
+    },
+    "key-2": {
+        "a": 3,
+        "b": 4,
+        "c": 5
+    }
+}
```

Comment:
You need to add newline at the end of file.

Reply:
Thanks for your comment! I'll fix it as soon as possible.